### PR TITLE
Add command description multilingual

### PR DESCRIPTION
**Pull request by 577fkj**
## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/main/java/emu/grasscutter/command/Command.java [hunk 1]
```diff
@@ -9,8 +9,6 @@
 
     String usage() default "No usage specified";
 
-    String description() default "No description specified";
```
Comment (4Benj) [inline src/main/java/emu/grasscutter/command/Command.java:12]: Instead of removing this you can make the description take the translation key instead

Comment (577fkj) [reply src/main/java/emu/grasscutter/command/Command.java:12]: OK, I'll change it back

Comment (577fkj) [reply src/main/java/emu/grasscutter/command/Command.java:12]: I added it back

**Review by ghost — CHANGES_REQUESTED** [review]
The description annotation should be setting the translation key, which you then retrieve with a description method to translate, instead of creating a description override for every command.

**Comment by 577fkj** [comment]
> The description annotation should be setting the translation key, which you then retrieve with a description method to translate, instead of creating a description override for every command.

Changed as required

**Review by ghost — APPROVED** [review]

**Pull request merged**